CONTRIBUTING.md

```diff
@@ -64,3 +64,30 @@ Then you just add a line to every git commit message:
 Use your real name (sorry, no pseudonyms or anonymous contributions.)
 
 If you set your `user.name` and `user.email` git configs, you can sign your commit automatically with `git commit -s`.
+
+## Contribution process
+
+### Development
+
+ - Create a personal fork of the project on GitHub.
+ - Use **sycl** branch as baseline for your changes.
+ - Prepare your patch (follow [LLVM coding standards](https://llvm.org/docs/CodingStandards.html)).
+ - Build the project and run all tests (see [GetStartedWithSYCLCompiler.md](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedWithSYCLCompiler.md))
+
+### Review and acceptance testing
+
+ - Create a pull request for your changes following [Creating a pull request instructions](https://help.github.com/articles/creating-a-pull-request/).
+ - Once the pull request is approved by an Intel representative, testing is started.
```

Comment:
Please, make it clear that **check_pr** is started w/o "Intel representative" approval.